test/integration/suites/oidc-discovery-provider/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM spire-agent:latest-local as spire-agent
+
+FROM oidc-discovery-provider:latest-local as oidc-discovery-provider
+
+FROM alpine:3.18.0 AS oidc-discovery-provider-mashup
+COPY --from=spire-agent /opt/spire/bin/spire-agent /opt/spire/bin/spire-agent
+COPY --from=oidc-discovery-provider /opt/spire/bin/oidc-discovery-provider /opt/spire/bin/oidc-discovery-provider
```

Comment:
`level=warning msg="Connection failed during accept" error="could not resolve caller information"`

Reply:
that may be because your agent container does not have enough permissions can you try with [pid](https://docs.docker.com/compose/compose-file/compose-file-v3/#pid)?
```
pid: "host"
```

Reply:
It works with this option, thanks!